src/account/operations/helpers/task.rs

```diff
@@ -0,0 +1,20 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+#[cfg(not(target_family = "wasm"))]
+pub(crate) fn spawn<F>(future: F) -> tokio::task::JoinHandle<F::Output>
+where
+    F: futures::Future + Send + 'static,
+    F::Output: Send + 'static,
+{
+    tokio::task::spawn(future)
+}
+
+#[cfg(target_family = "wasm")]
+pub(crate) fn spawn<F>(future: F) -> tokio::task::JoinHandle<F::Output>
+where
+    F: futures::Future + 'static,
+    F::Output: 'static,
+{
+    tokio::task::spawn_local(future)
+}
```

Comment:
I think I would put this file closer to the root as it's also being used in account_manager